.github/workflows/publish-hotfix.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```

Comment:
https://github.com/actions/setup-node provides a     node-version-file option we could reference .nvm from

Reply:
Added